src/main/scala/edu/knowitall/tool/srl/ClearSrl.scala

```diff
@@ -42,7 +42,8 @@ class ClearSrl extends Srl {
   def apply(graph: DependencyGraph): Seq[Frame] = {
     val tree = new DEPTree()
 
-    graph.nodes.zipWithIndex.foreach {
```

Comment:
I confirmed that the type being returned is inferred to be `Set`, not `SortedSet`:
```scala
// Patch, explicitly checking return type.
val withIndex: scala.collection.immutable.SortedSet[(DependencyNode, Int)] =
  graph.nodes.zipWithIndex
withIndex.foreach {
```
And then:
```
> compile
[error] (compile:compile) Compilation failed
[error] Total time: 2 s, completed Jun 2, 2016 10:30:40 AM
> testOnly edu.knowitall.openie.OpenIESpecTest
[info] Compiling 118 Scala sources to /Users/jkinkead/work/openie-standalone/target/scala-2.11/classes...
[error] /Users/jkinkead/work/openie-standalone/src/main/scala/edu/knowitall/tool/srl/ClearSrl.scala:53: type mismatch;
[error]  found   : scala.collection.immutable.Set[(edu.knowitall.tool.parse.graph.DependencyNode, Int)]
[error]  required: scala.collection.immutable.SortedSet[(edu.knowitall.tool.parse.graph.DependencyNode, Int)]
[error]     val withIndex: scala.collection.immutable.SortedSet[(DependencyNode, Int)] = graph.nodes.zipWithIndex
[error]                                                                                              ^
[error] one error found
```

I think it's more likely that the class load is affecting the default `hashCode` implementation, and changing the token sort that way.